chapters/BP_4014_en.md

```diff
@@ -0,0 +1,38 @@
+## Make sure that user journeys allow users to carry out their intended action
+
+### Identifiers
+
+| GreenIT | V2  | V3  |  V4  |
+|:-------:|:---:|:---:|:----:|
+|         |     |     |      |
+
+### Categories
+
+|    Life cycle    |    Tiers    |  Responsible   |
+|:----------------:|:-----------:|:--------------:|
+| 1. Specification | User/Device | UX/UI Designer |
+
+### Indications
+
+|      Priority      | Implementation difficulty | Ecological impact |
+|:------------------:|:-------------------------:|:-----------------:|
+|         4          |             4             |         4         |
+
+|                      Saved resources                      |
+|:---------------------------------------------------------:|
+|       Processor / RAM / Storage / Network / Queries       |
+
+### Description
+
+Web services allow talks like carrying out administrative procedures, opening contracts, claims declarations, etc. to be carried out without the need to travel to an office. Many of these procedures can be complex to carry out or may not take specific cases into account. For example, installing a mobile application, and therefore, having a smartphone may be required to validate the connexion to an online banking service. Therefore, some users may find themselves unable to use these services. Setting up more "low-tech" alternatives (telephone call, SMS, physical access point) can overcome this situation.
+
+### Example
+
+Set up a telephone number to help users during complex administrative procedures such as taking out an insurance policy in case of difficulties.
```

Comment:
```suggestion
Set up a phone number to help users during complex administrative procedures such as taking out an insurance policy in case of difficulties.
```